### PR TITLE
#1138 update integration test script with helm 3.0

### PR DESCRIPTION
**Pull request by glindsell**
#1138 

Signed-off-by: glindsell <gl@seldon.io>

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **adriangonz**

If they are not already assigned, you can assign the PR to them by writing `/assign @adriangonz` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
@glindsell: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 48a6ab3a9a2512c7eb4d042a93ac9510c7b7022a | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: testing/scripts/Makefile [hunk 1]
```diff
@@ -18,14 +18,10 @@ kind_build_operator:
 
 kind_build_images: kind_build_engine kind_build_operator
 
-install_helm:
```
Comment (axsaucedo) [inline testing/scripts/Makefile]: Installation tests fail as `install_helm` command is required, so either script needs to be modified or install_helm should be added

File: testing/scripts/Makefile [hunk 2]
```diff
@@ -18,14 +18,10 @@ kind_build_operator:
...
-	kubectl -n kube-system create sa tiller
-	kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
-	helm init --service-account tiller
-	kubectl rollout status deploy/tiller-deploy -n kube-system
-
 install_ambassador:
-	helm install stable/ambassador -f ambassador_values.yaml --name ambassador --set crds.keep=false --namespace seldon --set replicaCount=1
+	helm repo add stable https://kubernetes-charts.storage.googleapis.com/
```
Comment (axsaucedo) [inline testing/scripts/Makefile]: I do think the helm set up should be separate, as helm is used in other parts of the integration tests.

File: testing/scripts/Makefile [hunk 3]
```diff
@@ -18,14 +18,10 @@ kind_build_operator:
...
+	helm repo update
+	helm install ambassador stable/ambassador -f ambassador_values.yaml --set crds.keep=false --namespace seldon --set replicaCount=1
```
Comment (axsaucedo) [inline testing/scripts/Makefile]: Right now the end to end tests use the dependencies in the core-builder image, which I believe don't use helm 3.x, so this change would not only have to be done in the core-builder image, and you would have to change the image that it points towards

**Comment by glindsell** [comment]
Please see #1186

**Pull request closed without merging**